### ISSUE TITLE
Tag Selection error for first tag inside block

### ISSUE THREAD

**Issue by alexander-schranz** [sulu/sulu#5025]
| Q | A
| --- | ---
| Sulu Version | 2.0.3
| Browser Version | Firefox 72.0

# Actual Behavior

First tag is not used and get removed.

# Expected Behavior

Should not be removed when first loading the tags.

# Steps to Reproduce

```xml

        <block name="tag_blocks" default-type="tags">
            <meta>
                <title lang="de">Tags</title>
            </meta>

            <types>
                <type name="tags">
                    <properties>
                        <property name="tags" type="tag_selection">
                            <meta>
                                <title lang="de">Tags</title>
                            </meta>
                        </property>
                    </properties>
                </type>
            </types>
        </block>
```

![tags](https://user-images.githubusercontent.com/1698337/73559077-b30aea00-4454-11ea-9fa7-4101b5566317.gif)

**Comment by etidimjav** [comment]
I have almost the same error except that the tag is not saved at second time. I have to save initially the tags in settings to use them.
I'm using sulu 2.0.2 and tag_selection in one page template (not in block).
Thanks

**Comment by danrot** [comment]
After some digging I can tell that this problem only occurs when a new tag is added this way. If you use an already existing tag it is working as expected. For some reason the `MultiSelectionStore` tries to load the not yet existing tag from the server only if the property is within a block, which will return an empty response. The empty response is set as new value, and that's why the tag disappears immediately...

However, I didn't figure out yet, why this is only happening when the property is within a block...

**Comment by nnatter** [comment]
Not sure if this still happens. But if it does, it might be fixed by #5576 🙂

**Comment by nnatter** [comment]
Just tested this on `2.1.6`. The issue does not occur anymore 🎉
